app/uk/gov/hmrc/helplinefrontend/controllers/SelectNationalInsuranceServiceController.scala

```diff
@@ -42,8 +42,12 @@ class SelectNationalInsuranceServiceController @Inject()(implicit
     val result = SelectNationalInsuranceServiceForm.apply().bindFromRequest.fold(
       errors => BadRequest(selectNationalInsuranceService(errors)),
       {
-        case FindNiNumber => NotImplemented
-        case OtherQueries => NotImplemented
+        case FindNiNumber => request.session.get("HELPLINE_ORIGIN_SERVICE") match {
+          case Some("IV") => Redirect(s"${appConfig.findYourNationalInsuranceNumberFrontendUrl}/find-your-national-insurance-number/checkDetails?origin=IV")
+          case Some("PDV") => Redirect(s"${appConfig.findYourNationalInsuranceNumberFrontendUrl}/find-your-national-insurance-number/checkDetails?origin=PDV")
```

Comment:
can these two values
"IV" AND "PDV" be vals in the app config instead
i.e. 
// APP CONFIG
val IVOrigin: String = "IV"
val PDVOrigin: String = "PDV"
- amend the existing config value richard added to use these